src/debug/Kconfig

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config TELEMETRY
+	bool "enable telemetry"
+	default n
+	help
+	  Enable Telemetry
```

Comment:
If you really want to take a shortcut then do `rsource debug/telemetry/Kconfig` at the top-level but not this. Keep telemetry files in the telemetry subdirectory.

Reply:
Like this?